### DISCUSSION TITLE
Docker container broke after using the update script from 7.3 to 7.4 (before upgrade to PVE 8)

### DISCUSSION THREAD

**Discussion by bruno-dogancic** [tteck/Proxmox#1581] [Help Needed] [answered]
Dear @tteck thank you for great scripts!

I've just used the script to update PVE from 7.3 to 7.4 (as per this https://www.derekseaman.com/2023/06/how-to-proxmox-7-4-to-8-0-upgrade.html )

Everything went smooth, and all my containers work, except the docker one (in the beginning I used your script to setup this one a year ago or something). I cannot start the container, it throws this error:

```
run_buffer: 322 Script exited with status 1
lxc_init: 844 Failed to run lxc.hook.pre-start for container "104"
__lxc_start: 2027 Failed to initialize container "104"
TASK ERROR: startup for container '104' failed
```

My main idea now is not to complicate too much, but to retrieve few files --- a config for docker compose that worked to run Frigate as well as Frigate config.

But I cannot attach the container to access files....

```
root@pve:~# lxc-attach -n 104
lxc-attach: 104: ../src/lxc/attach.c: get_attach_context: 406 Connection refused - Failed to get init pid
lxc-attach: 104: ../src/lxc/attach.c: lxc_attach: 1470 Connection refused - Failed to get attach context
```

Do you have any suggestions?

Kind regards,
Bruno

**Comment by tteck** [comment]
I'd update the LXC https://github.com/tteck/Proxmox/discussions/1498

**Reply by bruno-dogancic** [reply]
Thank you for the reply mate, youa re fast! :)

**Reply by bruno-dogancic** [reply]
I have no time nor nerves atm to tweak it, but I might do so when I get time

**Comment by bruno-dogancic** [comment, marked as answer]
Hmmm, a simple restore of backup (that I luckily had after tweaking working confings!) resolved the problem
